cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -427,7 +427,8 @@ CAST_VARLEN_TYPE_FROM_NUMERIC(VARBINARY)
 #undef GDV_FN_CAST_VARCHAR_REAL
 
 GANDIVA_EXPORT
```

Comment:
GANDIVA_EXPORT on helper functions is I think not required

Reply:
I removed it